indigo_app/templates/indigo_api/work_form.html

```diff
@@ -338,6 +338,20 @@ <h5 class="card-header">Repeal</h5>
       </div>
     </div>
     {% endblock work-form-repeal %}
+    <div class="form-row">
+      <div class="form-group col-md-4">
+        <label for="{{ form.as_at_date.id_for_label }}">{{ form.as_at_date.label }}</label>
+        <input type="text" class="form-control" data-provide="datepicker" id="{{ form.as_at_date.id_for_label }}" placeholder="yyyy-mm-dd" pattern="\d{4}-\d\d-\d\d" name="{{ form.as_at_date.html_name }}" value="{{ form.as_at_date.value|default_if_none:''|stringformat:'s' }}">
+        <p class="form-text text-muted">Date up to which this work was last checked for updates.</p>
```

Comment:
I think this needs to refer to the fact that if it's blank, this will be computed.

`Date up to which this work was last checked for updates. If unset, uses the greater of {{ place.name }}'s as-at date, and this work's latest expression date.`